### ISSUE TITLE
Feature request: MKDocs for documentation and tutorials

### ISSUE THREAD

**Issue by stephanbisser** [pnp/generator-teams#238]
### 💡 Idea

I'd like to see tutorials and docs on a dedicated GH page using MKDocs similar to other PnP initiatives

### Is your feature related to a bug

no

**Comment by appieschot** [comment]
Coming from CLI I think we should aim for a V1 and a V2. V1 the basics, V2 with tutorials. 

- Home (basic intro) 
- User Guide
	- Installing
	- Updating
- Concepts 
	- Generator structure
	- What's part of our getting started etc. 
- About 

I guess we should also use the GitHub wiki for the Minimal Path to Awesomeness

**Comment by appieschot** [comment]
Thinking about it a bit more we should also 'move' and upgrade our current docs to here as well. I can imagine that we have three steps: 

- V1 the basics
- V2 move and restructure current docs
- V3 tutorials 

I would suggest to use this issue for the V1 💪 once that is in place we could create a seperate issue for the move of current docs, and for each tutorial a new issue.

**Comment by stephanbisser** [comment]
Yes, I'd also go with Github Pages for docs and tutorials only as this has a better layout than just .md files in the repo directly. Similar to what the CLI docs look like right?

**Comment by pkbullock** [comment]
There is already a site in place - https://pnp.github.io/generator-teams/ just needs to be upgraded and completed. The tech used by most of the PnP Repos is MKDocs (CLI, SPFX Samples) and DocFx (PnP Core, PowerShell, Script Samples) this one is MKDocs based. so just need to copy the theme files, maybe use a theme colour linking to the team logo, and setup github actions for automation.

**Comment by RickVanRousselt** [comment]
I will take this one.

**Comment by RickVanRousselt** [comment]
First draft is ready. Took V1 and a little of V2 😉 And empty site is only so much :-) Let me know what you think. I put it on my[ GitHub pages](https://rickvanrousselt.github.io/) temporarily not to break the current one.

Some things I would love to get some input on:
- Used parker both as a logo and as a favicon
- Merged all the separate release notes into 1 file already but there is still something not right, but I just cannot put my finger on it.
- We should review the user guides (I already added some links to MS docs) but we might need another issue to discuss what to keep and where we reference MS docs
- Google Analytics: I currently just re-used one of my ID's to test. We will need a real one, and MKDocs cookie banner is still in preview so we are non-GDPR compliant :-)

**Comment by wictorwilen** [comment]
Love it, looks great. 

Re GA - does MKDocs support App Insights? That's what's being used for the generator telemetry, and why not reuse that?

**Comment by stephanbisser** [comment]
Looks great @RickVanRousselt !
I'd say we use Parker for both.
We should definitely discuss which user guides and tutorials we need in the docs.

**Comment by appieschot** [comment]
Sounds like we should get this release out there asap and go from there 🚀

**Comment by appieschot** [comment]
Since we have our first release ready we will close this issue. New requests can be added through new issues 🚀
